### ISSUE TITLE
SC Glyph Improvement Suggestion involving 乌

### ISSUE THREAD

**Issue by starkshaw** [adobe-fonts/source-han-serif#64]
### Prerequisites

* [x] If you are reporting an issue that affects glyphs for characters for a particular region or regions, did you verify that the characters are within the supported scope of the region or regions? This generally means GB 18030 or Tōngyòng Guīfàn Hànzìbiǎo (通用规范汉字表) for China, Big Five or CNS 11643 Planes 1 & 2 for Taiwan, HKSCS-2016 for Hong Kong (though support has been deferred until Version 2.000), the JIS standards for Japan, and KS X 1001 and KS X 1002 for Korea.
* [x] Did you thoroughly search the [open](https://github.com/adobe-fonts/source-han-serif/issues) and [closed](https://github.com/adobe-fonts/source-han-serif/issues?q=is%3Aissue+is%3Aclosed) issues to avoid reporting a duplicate issue?
* [x] Did you go through the [official font readme file](https://github.com/adobe-fonts/source-han-serif/raw/release/SourceHanSerifReadMe.pdf) to better understand the scope of the project, to include the *Known Issues* section at the very end?

### Description

I found that the upper box in 乌 is fairly wider and bigger than conventional fonts. Sometimes it looks odd when the font size is larger. Here are some comparisons of Noto Sans CJK SC with SinoType Songti SC in different weights, note that the issue may not be very evident in some weights, but very clear in Regular:

<img width="450" alt="screen shot 2017-06-09 at 03 56 18" src="https://user-images.githubusercontent.com/9618357/26959457-2a1739c8-4cc8-11e7-80da-00df4f95eb27.png">
<img width="450" alt="screen shot 2017-06-09 at 03 56 32" src="https://user-images.githubusercontent.com/9618357/26959459-2a2c11b8-4cc8-11e7-99b1-ae3d0f45fba2.png">
<img width="450" alt="screen shot 2017-06-09 at 03 56 43" src="https://user-images.githubusercontent.com/9618357/26959460-2a30779e-4cc8-11e7-95c5-ea12daf8e3f8.png">
<img width="450" alt="screen shot 2017-06-09 at 03 56 55" src="https://user-images.githubusercontent.com/9618357/26959458-2a2a7f42-4cc8-11e7-8a51-99725a84dd25.png">

I understand it might be a matter of style but I also believe it might be something need extra attention because 乌 and its derivatives are only used in simplified Chinese. Please consider a design that could make the box looked not so *blank*.

**Comment by kenlunde** [comment]
You are comparing two completely different and unrelated typeface designs.

**Comment by hfhchan** [comment]
This could be considered as part of a general "The counters of many CN and TW glyphs for ideographs are too wide." in #36.

**Comment by starkshaw** [comment]
Dear @kenlunde 

I am fully aware of the fact that they are different typefaces. All I am trying to say is since SinoType Songti is maybe one of the most popular and familiar Songti (Serif) typeface among all simplified Chinese users. And there is no such Noto Serif CJK SC alternatives to simplified Chinese users, unlike Kozuka Mincho or Hiragino Mincho to Japanese users, therefore SC users are not familiar with such design pattern, and sooner or later they will find some flaws that are not fit into their conventional sense of aesthetics, such as this one. Like I said, this issue I opened maybe solely a matter of style, and it is following the *standard*, but quality is always above the standard, and that is the reason why I opened this issue. Please reconsider my proposal, thanks.

**Comment by kenlunde** [comment]
I wholeheartedly disagree. You are steering the design direction of a particular typeface to become that of another typeface, which is never a good idea. We are aware that the glyphs for characters outside the scope of GB 2312 have wide counter space, but that is a completely separate and independent issue, and something that we plan to address longer term.

**Comment by Explorer09** [comment]
I'm sorry to bump this, but I don't understand the problem of why @starkshaw would think the counter of 乌 component looks so "blank" in Source Han. I've looked at the glyphs of 鸟乌邬鸵坞呜鳥烏鳴 in Source Han Sans & Serif and didn't quite get the idea of what could go wrong. Personally I think the 鸟 (bird) component looks fine (i.e. balanced), but I cannot say the same about 乌 (black/crow).
So, may I ask a few questions to starkshaw: Does your impression of large counter only happens with 乌, or it does with 鸟 as well? If it's the former, then I can suggest something. Note that I'm not telling you to compare with SinoType Songti or other typefaces; I only wish to address the balancing problem of Source Han itself.

**Comment by hfhchan** [comment]
As a Traditional Chinese user who also regularly reads Simplified Chinese, I find the character 乌 and 乌 only has a noticeable counter too large.

I also find characters above with the 乌 component "strange" when they are stacked beside 鸟, as the visual counter for 乌 appears larger than that for 鸟, especially for the Light weight.

However, they look "fine" with other characters, because the counters of characters in Source Han Serif is wide anyway.

**Comment by Explorer09** [comment]
@hfhchan
From what I observed, the wide counter of Source Han is not the problem, but the relative positioning and spacing could be.

My suggestion is like this. Allow me to highlight the spacing ("apertures" in typography) and the distances of two hook (㇆) strokes:
![bird-radical](https://user-images.githubusercontent.com/2487263/26979975-4cd31f18-4d63-11e7-8568-6dd8523a78a6.png)

Now, I said 鸟 looks fine, because the "bird eye" (㇔) justifies the _wide_ counter of the "bird head" (㇆).

But for 乌, the "crow's eye" is not visible, and there will look like a large white area in the center. For traditional 烏, this is balanced by giving the crow a shorter (and flatter) head (コ). However, the "bird head" simplified 鸟 and 乌 can look rather squarish because the "eye" is a dot (㇔) stroke and not horizontal (㇐). Therefore, with the eye removed, the head (㇆) in simplified 乌 should be made slightly narrower to balance the white area.

In short, I think it should be c₂ > c₁ (slightly, to balance the space) on the picture, not c₂ = c₁.

Also, it should be a₂ < a₁. A closer _aperture_ for a wide counter with nothing inside. This can be done by extending the hook (勾) to slightly more left, or by narrowing the head (㇆) and moving the hook left along the way. (SinoType Songti did a right balance on this part by placing the hook (勾) in a rather center position.)

b₂ > b₁. Source Han already did this part right.

**Comment by starkshaw** [comment]
@kenlunde I am sorry that you misunderstood me. I am not comparing these two typefaces to ask you matching the design of another font. I did this because I want to show how big that space is compare to a daily-use font. I can't just say *it looks blank* without proof.

**Comment by starkshaw** [comment]
@Explorer09 鸟 is used to compare with 乌. And yes 鸟 is fine.

**Comment by starkshaw** [comment]
@hfhchan I can see your point. I discovered this issue by accidentally enlarged a PDF in this typeface and saw a medium weight word 好莱坞 (Hollywood) and found that 乌 in 坞 looks extra odd. 乌 is narrowed since the presence of 土, but not well balanced of the blank space.
